tests/engine/test_flow_runner.py

```diff
@@ -708,6 +708,16 @@ def test_run_count_tracked_via_retry_states(self):
         assert isinstance(state3.result[t2], Failed)
 
 
+def test_flow_runner_uses_default_executor_on_flow_if_present():
+    t = SuccessTask()
+    with Flow(name="test", executor=Executor()) as flow:
+        result = t()
+
+    with raise_on_exception():
+        with pytest.raises(NotImplementedError):
+            FlowRunner(flow=flow).run(executor=Executor())
```

Comment:
```suggestion
            FlowRunner(flow=flow).run(executor=None)
```
otherwise you don't know whether it was using the passed in Executor class or the Flow executor class, right?

Reply:
Oop, good catch. Fixed.